data/INRA/data6/2016ALFColomb_transf_parsed.ttl

```diff
@@ -0,0 +1,6843 @@
+
```

Comment:
don't you think it's better to first fix and check the existing files before adding more files? I'm not going to check new files before we make sure the existing files are ok